packages/react-api/src/api/SQL.js

```diff
@@ -110,15 +108,17 @@ function createRequest({
   });
 
   const isGet = getUrl.length < REQUEST_GET_MAX_URL_LENGTH;
-  if (isGet) {
+  if (isGet && apiVersion !== API_VERSIONS.V3) {
     return getRequest(getUrl, requestOpts);
   }
+  if (isGet && apiVersion === API_VERSIONS.V3) {
+    return getRequest(getUrl, requestOpts, {
+      Authorization: `Bearer ${credentials.accessToken}`
+    });
+  }
```

Comment:
This could be a little bit more readable:

```
if (isGet) {
    if (apiVersion === API_VERSIONS.V3) {
        // request with header
    } else {
        // request without header
    }
}
```